python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -811,6 +812,9 @@ def do_evaluate(
                     else prefix.obj,
                 )
             )
+        elif self.name == pl_expr.StringFunction.Strptime:
```

Comment:
@wence- I'm a little unsure if what's being passed from rust here is correct. The [docs](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.str.strptime.html) list many options for this api but `self.options` is empty here. [the rust code](https://github.com/pola-rs/polars/blob/ebba58da86bc575912613068eba0b6613f9d4226/py-polars/src/lazyframe/visitor/expr_nodes.rs#L872) only passes the enum value out, is this right?

Reply:
Yes, we didn't send that through in https://github.com/pola-rs/polars/pull/16221.

Please check if https://github.com/pola-rs/polars/pull/17702 works for you

Reply:
seems to be working, with that PR I get this:
```
(Pdb++) self.options
('%Y-%m-%d', True, True, True)
```